src/state/loading/types.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line
+export const TOGGLE_LOADING = 'loading/set-references-loading';
```

Comment:
why the action type is "set-references-loading" ? IMO should be "toggle-loading"